config/default.yml

```diff
@@ -6,6 +6,12 @@ RSpec:
   Include:
     - "**/*_spec.rb"
     - "**/spec/**/*"
+  Exclude:
+    - "**/spec/factories.rb"
+    - "**/spec/factories/**/*.rb"
+    - "**/test/factories.rb"
+    - "**/test/factories/**/*.rb"
+    - "**/features/support/factories/**/*.rb"
```

Comment:
I am not sure that we need to exclude files that we haven’t asked RuboCop to include.

Reply:
This roughly means “include spec/**/*, but not spec/factories”.

Reply:
Fair enough regarding 12-13

But we do include spec/features eg  https://github.com/mastodon/mastodon/blob/dc7e2985e60c90e8efe82e56edb4fdd473bd1651/spec/features/profile_spec.rb

They may have spec/features/support/factories which we don’t want to expect.

Reply:
Good point.